### DISCUSSION TITLE
help / installation paquets  RPM

### DISCUSSION THREAD

**Discussion by guitoube95** [community/community#37126] [Other Feature Feedback, Questions, & Ideas]
Bonjour tout le monde !

voila je débute tout juste dans le monde de la programmation mais je suis franchement perdu .

 je débute avec le site  https://nodeschool.io/fr-fr/

sauf que je suis perdu pour commencer les ateliers autrement dit installer les paquets NPM ,  je n'y arrive pas du tout !

J'ai regardé pas mal de tutoriel mais je n'ai pas réussi

aidez moi s'il vous plait

merci par avance

**Comment by ossanna16** [comment]
Hi there @guitoube95 and welcome to our community! Thank you for asking a great question 🙂
